URLSessionStartProject/Sources/Scenes/Controller/Helpers/CardsPrinter.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CardsPrinter.swift
+//  URLSessionStartProject
+//
+//  Created by Михаил Багмет on 24.05.2022.
+//
+
+import Foundation
+
+class CardsPrinter {
```

Comment:
Помнишь рекомендацию Apple? 😁
> Use structures by default
```suggestion
struct CardsPrinter {
```

Reply:
Глянул материал по выбору между структурой и классом. Перечитаю все более подробно, чтобы наложилось на текущий опыт.